scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -138,53 +150,137 @@ workflow GvsCreateFilterSet {
+                one_thousand_genomes_resource_vcf = one_thousand_genomes_resource_vcf,
+                one_thousand_genomes_resource_vcf_index = one_thousand_genomes_resource_vcf_index,
+                dbsnp_resource_vcf = dbsnp_resource_vcf,
+                dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
+                use_allele_specific_annotations = true,
+                disk_size = small_disk
+        }
 
-   call UploadFilterSetToBQ {
-     input:
-        gatk_override = gatk_override,
-        filter_set_name = filter_set_name,
+        scatter (idx in range(length(ExtractFilterTask.output_vcf))) {
+            call SNPsVariantRecalibrator as SNPsVariantRecalibratorScattered {
+                input:
+                    sites_only_variant_filtered_vcf = ExtractFilterTask.output_vcf[idx],
+                    sites_only_variant_filtered_vcf_index = ExtractFilterTask.output_vcf_index[idx],
+                    recalibration_filename = filter_set_name + ".snps." + idx + ".recal",
+                    tranches_filename = filter_set_name + ".snps." + idx + ".tranches",
+                    recalibration_tranche_values = snp_recalibration_tranche_values,
+                    recalibration_annotation_values = snp_recalibration_annotation_values,
+                    model_report = SNPsVariantRecalibratorCreateModel.model_report,
+                    hapmap_resource_vcf = hapmap_resource_vcf,
+                    hapmap_resource_vcf_index = hapmap_resource_vcf_index,
+                    omni_resource_vcf = omni_resource_vcf,
+                    omni_resource_vcf_index = omni_resource_vcf_index,
+                    one_thousand_genomes_resource_vcf = one_thousand_genomes_resource_vcf,
+                    one_thousand_genomes_resource_vcf_index = one_thousand_genomes_resource_vcf_index,
+                    dbsnp_resource_vcf = dbsnp_resource_vcf,
+                    dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
+                    use_allele_specific_annotations = true,
+                    disk_size = small_disk
+            }
+        }
 
-        sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
-        sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
+        call Tasks.GatherTranches as SNPGatherTranches {
+            input:
+                tranches = SNPsVariantRecalibratorScattered.tranches,
+                output_filename = filter_set_name + ".snps.gathered.tranches",
+                mode = "SNP",
+                disk_size = small_disk,
+                gatk_override = gatk_override
+        }
 
-        snp_recal_file = SNPsVariantRecalibrator.recalibration,
-        snp_recal_file_index = SNPsVariantRecalibrator.recalibration_index,
-        snp_recal_tranches = SNPsVariantRecalibrator.tranches,
+        scatter (idx in range(length(ExtractFilterTask.output_vcf))) {
+            call CreateFilterSetFiles as CreateFilterSetFilesScattered {
```

Comment:
I think this doesn't do what we want… the sites_only variants are going to be loaded multiple times (one for each scatter).  Without going too far down the rabbit hole, maybe the thing to do here is to Merge the VCFs coming out of the scatter and then load them the same way we do in the classic mode?